From b758fd588312152cfa9874696cd7ed2c6147a93b Mon Sep 17 00:00:00 2001
From: "qingzong.ma" <qingzong.ma@quectel.com>
Date: Mon, 13 Nov 2023 10:11:05 +0800
Subject: [PATCH 042/188] wifi: ath11k: remove extra warning message, move to
 debug

(cherry picked from commit fed22e31117d38ceaac59fec4a369cbf5806ab86)
Signed-off-by: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
---
 drivers/net/wireless/ath/ath11k/dp_rx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath11k/dp_rx.c b/drivers/net/wireless/ath/ath11k/dp_rx.c
index 8e2abc7b8..ca8e667ec 100644
--- a/drivers/net/wireless/ath/ath11k/dp_rx.c
+++ b/drivers/net/wireless/ath/ath11k/dp_rx.c
@@ -2560,7 +2560,7 @@ static int ath11k_dp_rx_process_msdu(struct ath11k *ar,
 	lrx_desc = (struct hal_rx_desc *)last_buf->data;
 	rx_attention = ath11k_dp_rx_get_attention(ab, lrx_desc);
 	if (!ath11k_dp_rx_h_attn_msdu_done(rx_attention)) {
-		ath11k_warn(ab, "msdu_done bit in attention is not set\n");
+		ath11k_dbg(ab, ATH11K_DBG_DATA, "msdu_done bit in attention is not set\n");
 		ret = -EIO;
 		goto free_out;
 	}
-- 
2.55.0

